src/less/tooltip/base/tooltip.less

```diff
@@ -52,20 +52,22 @@ button.tooltip__close {
 
 .tooltip__pointer {
     .pointer-base();
-    .pointer-top();
     .pointer-center();
     .customBackgroundColorProperty(tooltip-background-color);
 }
 
 .tooltip__pointer--top-left {
+    .pointer-top();
     .pointer-top-left();
 }
 
 .tooltip__pointer--top {
+    .pointer-top();
     .pointer-top-center();
 }
 
 .tooltip__pointer--top-right {
+    .pointer-top();
```

Comment:
Hmm, the `.pointer-top-right()` mixin under bubble-mixins already includes the `.pointer-top();` mixin, so this shouldn't be necessary (and was working fine up until some point). Can you see what is going on there?

Reply:
Here is the code reference: https://github.com/eBay/skin/blob/1401-tooltip-wrong-direction/src/less/mixins/bubble/base/bubble-mixins.less#L98

Reply:
Yes, the issue is that the `tooltip__pointer` class had the `.pointer-top()` mixin. That was interfering with the `tooltip-pointer--bottom-left` and `tooltip-pointer--bottom` classes, because the `top: auto` property wasn't overriding the `top: -7px;` from the `pointer-top` like it should have been. 

So I guess another strategy would have been changing the specificity of the classes for the `bottom` and bottom`left` classes... but this seemed a little more straightforward.

Reply:
What I did here was take out the pointer-top() mixin from the tooltip__pointer class and then added it in to the tooltip__pointer--top classes to make sure it was there, as it was previously applying to them from the `tooltip__pointer` class.